src/lib/components/Radio/README.mdx

```diff
@@ -77,6 +77,11 @@ See [API](#api) for all available options.
 - Use **clear, calm error messages** when there's a problem with what they
   entered.
 
+- In the background, InputGroup uses the [`fieldset`][fieldset] element. Not
```

Comment:
Do we want to communicate this to the user? 

Isn't it an implementation detail? Are there any cases when using div as before would be preferable from the users perspective? If not, then I don't think we should bother the user with this information.

Reply:
> … it also allows you to make use of its built-in features like disabling all nested inputs or pairing the group with a form outside.

I don't think you want to use a `<div>` instead, but this definitely is a benefit.